src/item.py

```diff
@@ -34,9 +34,9 @@ class DataItem(Graphs.Item):
     markerstyle = GObject.Property(type=int, default=0)
     markersize = GObject.Property(type=float, default=7)
 
-    @staticmethod
-    def new(params, xdata=None, ydata=None, **kwargs):
-        return DataItem(
+    @classmethod
```

Comment:
The main reason, was that classmethods can be inherited. Say we introduce `DataItemButWithInternalHistory` which doesn't need its own new method, since it has no new properties to render. using the staticmethod we would only get a `DataItem` whereas a classmethod will indeed get the correct type.